src/deploy/jenkins-uat.js

```diff
@@ -0,0 +1,22 @@
+const fetch = require('node-fetch');
+
+module.exports = () => {
+    const jenkinsUrl = "https://ctv-jenkins-02.truex.com/generic-webhook-trigger/invoke";
+    return fetch(jenkinsUrl, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json",
+            "token": "TAR-HTML5_UAT",
+        },
+        body: JSON.stringify({
+            "whakapapaBranchName": "develop",
+            "devicePlatforms": "Unknown,Tizen,LG,XboxOne,FireTV,AndroidTV,Vizio,Comcast",
+            "skylineBranchName": "develop",
+            "skylineS3BranchDirName": "develop",
+            "fromBuildJobUrl": process.env.TRAVIS_BUILD_WEB_URL,
+            "sourceRepo": process.env.TRAVIS_REPO_SLUG
```

Comment:
we should figure out if we want to use env vars here, or make these function args
I am inclined to make them function args, so that we can test this in isolation
the call can supply the args from the env vars instead

Reply:
I'll make it arguments since I'm doing it for other things.  The isolation testing is a good byproduct as well and even allows potential for manual invocation